datafusion/core/tests/parquet_pruning.rs

```diff
@@ -237,7 +237,7 @@ async fn prune_int32_scalar_fun() {
     test_prune(
         Scenario::Int32,
         "SELECT * FROM t where abs(i) = 1",
-        Some(4),
+        Some(0),
```

Comment:
The changes in this file are due to the fact that these predicates no longer error (instead they evaluate successfully to all `true`,  meaning all record groups are kept)

The `3` is the number of rows that pass this predicate -- so no actual answers are changed here